neutron_lbaas_dashboard/static/dashboard/project/lbaasv2/workflow/certificates/create-certificate.controller.js

```diff
@@ -0,0 +1,346 @@
+/*
+ *    (c) Copyright 2015 Hewlett-Packard Development Company, L.P.
```

Comment:
The copyright of this new file should be F5, instead of HP, correct?

Reply:
done, please review.